app/views/customers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Name: <%= @customer.name %> </h1>
+<h3> Add Items: </h3> 
+<%= form_with url: "/customers/#{@customer.id}", method: :get, local: true do |form| %>
```

Comment:
Use white space between sections to help with readability of html elements, so a white line after line 2 would be helpful.